### ISSUE TITLE
Feature request: Add support for mocking getIdTokenResult.

### ISSUE THREAD

**Issue by simplecohesion** [atn832/firebase_auth_mocks#59]
Most useful for custom claims, which can be used for role-based access control.

``` dart 
  /// Returns a [IdTokenResult] containing the users JSON Web Token (JWT) and
  /// other metadata.
  ///
  /// If [forceRefresh] is `true`, the token returned will be refreshed regardless
  /// of token expiration.
  Future<IdTokenResult> getIdTokenResult([bool forceRefresh = false]) {
    return _delegate.getIdTokenResult(forceRefresh);
  }
```

**Comment by atn832** [comment]
Fixed by #73. I'll publish an update in the next few days.

**Comment by atn832** [comment]
Just published the fix to https://pub.dev/packages/firebase_auth_mocks/changelog#087.
